### PR TITLE
Leaf/a4.8 http post call handling

### PR DESCRIPTION
**Pull request by K0725**
Task Post call Handling for interest tag #21 

- Implemented HTTP Post call handling for tag field.

### CODE REVIEW

**Review by JRSiwiecki — CHANGES_REQUESTED** [review]
Your code looks mostly fine, however it seems that your branch has changes from other users that weren’t assigned this issue, which should be fixed, as per the announcement on 06/26/23 in #sprint-announcements on the discord.

This could be fixed in a few ways, but the simplest way could be creating a new branch off of dev_new, git cherry-picking your commits only, deleting this current branch, and then requesting a new pull request.

File: mesh/tags/views.py [hunk 1]
```diff
@@ -35,3 +35,35 @@ def user_tags(request):
...
+    if request.method == "POST":
+        data = request.POST
+        required_fields = ["tagName", "isDefault", "accountID"]  
+        if not all(field in data for field in required_fields):
+            response_data = {'status': "Error: Missing field"}
+            return JsonResponse(response_data, status=400)
+        
+        try:
+            # Creating new tag object
+            tag = Tag.objects.create(
```
Comment (GuyWhoCode) [inline mesh/tags/views.py:50]: You're missing an import here

File: mesh/tags/views.py [hunk 2]
```diff
@@ -35,3 +35,35 @@ def user_tags(request):
...
         return JsonResponse(response_data, status=200)
+
+
+def post_tags(self, request):
+    if request.method == "POST":
+        data = request.POST
+        required_fields = ["tagName", "isDefault", "accountID"]  
+        if not all(field in data for field in required_fields):
+            response_data = {'status': "Error: Missing field"}
+            return JsonResponse(response_data, status=400)
```
Comment (GuyWhoCode) [inline mesh/tags/views.py:46]: Have you tested your code with Postman? When I tried making a post request, I'm unable to get a response when there are no fields.

**Review by GuyWhoCode — CHANGES_REQUESTED** [review]
Please re-request my review after you've taken a look at my comments. Also, change `settings.py` to not include an import to the `dotenv` library,

File: mesh/tags/views.py (hunk 1 above)
Comment (K0725) [reply mesh/tags/views.py:50]: ohh I see

File: mesh/tags/views.py (hunk 2 above)
Comment (K0725) [reply mesh/tags/views.py:46]: I've tested the postman but I just created my own test cases. So I will get back to on this one

**Pull request closed without merging**